src/guiFormSpecMenu.cpp

```diff
@@ -2728,13 +2733,14 @@ void GUIFormSpecMenu::acceptInput(FormspecQuitMode quitmode=quit_mode_no)
 					// without rtti support in irrlicht
 					IGUIElement * element = getElementFromId(s.fid);
 					gui::IGUIComboBox *e = NULL;
+					s32 selected = -1;
 					if ((element) && (element->getType() == gui::EGUIET_COMBO_BOX)) {
 						e = static_cast<gui::IGUIComboBox*>(element);
+						selected = e->getSelected();
 					}
-					s32 selected = e->getSelected();
 					if (selected >= 0) {
-						fields[name] =
-							wide_to_utf8(e->getItem(selected));
+						fields[name] = wide_to_utf8(
+							e->getItem(selected));
```

Comment:
Well, depends on how many chars you count tabs as.  With the official 8 it's 99 chars.